src/Ledger/Utxo/Properties.lagda

```diff
@@ -8,24 +8,25 @@ open import Ledger.Transaction
 
 module Ledger.Utxo.Properties (txs : TransactionStructure) where
 
-open import Prelude
-open import Ledger.Prelude
+open import Prelude hiding (_+_)
+open import Ledger.Prelude renaming (_+_ to _+ℕ_)
+
+open import Algebra              using     (CommutativeMonoid ; Monoid)
+open import Algebra.Morphism     using     (module MonoidMorphisms ; IsMagmaHomomorphism)
+open import Data.Integer         using     (ℤ; _⊖_ ; _-_ ; 0ℤ)
+                                 renaming  (_+_ to _+ℤ_ ; +_ to _⁺ ; -_ to _⁻)
+open import Data.Integer.Ext     using     (posPart ; negPart)
+open import Data.Nat.Properties  hiding    (_≟_)
+open import Data.Sign            using     (Sign)
+open import Relation.Binary      using     (IsEquivalence)
```

Comment:
It was hard for me to carry out the revision/upgrade to implicit instances without simplifying and organizing the imports.  The use of instances like the `Add` record used below naturally cause conflicts with other imports, but this was certainly not the only name clash I encountered while working on this PR. 

Not only will fewer clashes occur if we are more careful/disciplined about imports, but also I find it much easier/faster to resolve such conflicts if the imports are well-organized, explicit, and as small as possible.

Reply:
reverted